2023-01-13 11:54:36 jdbc[3]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1573)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:470)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:360)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:712)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:587)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:828)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:705)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:704)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
2023-01-13 11:55:56 jdbc[3]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1573)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:470)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:360)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:712)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:587)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:828)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:705)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:704)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
2023-01-13 11:56:36 jdbc[3]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1573)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:470)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:360)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:712)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:587)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:828)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:705)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:704)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
2023-01-13 11:56:36 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "REQUESTS" не найдена
Table "REQUESTS" not found; SQL statement:
CREATE TABLE IF NOT EXISTS items
(
    id          BIGINT GENERATED BY DEFAULT AS IDENTITY NOT NULL,
    name        VARCHAR(255),
    description VARCHAR(4000),
    available   BOOLEAN,
    owner_id    BIGINT                                  NOT NULL,
    request_id  BIGINT,
    CONSTRAINT pk_item PRIMARY KEY (id),
    CONSTRAINT FK_ITEM_ON_OWNER FOREIGN KEY (owner_id) REFERENCES users (id),
    CONSTRAINT FK_ITEM_ON_REQUEST FOREIGN KEY (request_id) REFERENCES requests (id),
    CONSTRAINT UQ_OWNER_ITEM_NAME UNIQUE (owner_id, name)
) [42102-210]
2023-01-13 11:56:36 jdbc[4]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1573)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:470)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:360)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:712)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:587)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:828)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:705)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:704)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
2023-01-13 11:57:33 jdbc[5]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1573)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:470)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:360)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:712)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:587)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:828)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:705)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:704)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
2023-01-13 12:36:06 jdbc[3]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1573)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:470)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:360)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:712)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:587)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:828)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:705)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:704)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
2023-01-13 12:36:06 jdbc[4]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1573)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:470)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:360)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:712)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:587)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:828)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:705)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:704)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
2023-01-13 12:36:11 jdbc[5]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1573)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:470)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:360)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:712)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:587)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:828)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:705)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:704)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
2023-01-13 12:37:09 jdbc[3]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1573)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:470)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:360)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:712)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:587)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:828)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:705)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:704)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
2023-01-13 12:37:09 jdbc[4]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1573)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:470)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:360)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:712)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:587)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:828)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:705)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:704)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
2023-01-16 09:47:08 database: flush
org.h2.message.DbException: Внутренняя ошибка: "org.h2.mvstore.MVStoreException: The file is locked: C:/Users/komov/dev/java-shareit/db/shareit.mv.db [2.1.214/7]"
General error: "org.h2.mvstore.MVStoreException: The file is locked: C:/Users/komov/dev/java-shareit/db/shareit.mv.db [2.1.214/7]" [50000-214]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.message.DbException.convert(DbException.java:395)
	at org.h2.mvstore.db.Store.lambda$new$0(Store.java:125)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:3318)
	at org.h2.mvstore.MVStore.panic(MVStore.java:593)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:469)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:4082)
	at org.h2.mvstore.db.Store.<init>(Store.java:136)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:138)
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:364)
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:206)
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:476)
	at com.zaxxer.hikari.pool.HikariPool.checkFailFast(HikariPool.java:561)
	at com.zaxxer.hikari.pool.HikariPool.<init>(HikariPool.java:115)
	at com.zaxxer.hikari.HikariDataSource.getConnection(HikariDataSource.java:112)
	at org.springframework.jdbc.datasource.DataSourceUtils.fetchConnection(DataSourceUtils.java:159)
	at org.springframework.jdbc.datasource.DataSourceUtils.doGetConnection(DataSourceUtils.java:117)
	at org.springframework.jdbc.datasource.DataSourceUtils.getConnection(DataSourceUtils.java:80)
	at org.springframework.jdbc.datasource.init.DatabasePopulatorUtils.execute(DatabasePopulatorUtils.java:52)
	at org.springframework.boot.jdbc.init.DataSourceScriptDatabaseInitializer.runScripts(DataSourceScriptDatabaseInitializer.java:90)
	at org.springframework.boot.sql.init.AbstractScriptDatabaseInitializer.runScripts(AbstractScriptDatabaseInitializer.java:145)
	at org.springframework.boot.sql.init.AbstractScriptDatabaseInitializer.applyScripts(AbstractScriptDatabaseInitializer.java:107)
	at org.springframework.boot.sql.init.AbstractScriptDatabaseInitializer.applySchemaScripts(AbstractScriptDatabaseInitializer.java:97)
	at org.springframework.boot.sql.init.AbstractScriptDatabaseInitializer.initializeDatabase(AbstractScriptDatabaseInitializer.java:75)
	at org.springframework.boot.sql.init.AbstractScriptDatabaseInitializer.afterPropertiesSet(AbstractScriptDatabaseInitializer.java:65)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.invokeInitMethods(AbstractAutowireCapableBeanFactory.java:1863)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1800)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:620)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:542)
	at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:335)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:234)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:333)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:208)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:322)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:208)
	at org.springframework.context.support.AbstractApplicationContext.getBean(AbstractApplicationContext.java:1154)
	at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:908)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:583)
	at org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext.refresh(ServletWebServerApplicationContext.java:147)
	at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:734)
	at org.springframework.boot.SpringApplication.refreshContext(SpringApplication.java:408)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:308)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1306)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1295)
	at ru.practicum.shareit.ShareItApp.main(ShareItApp.java:10)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: Внутренняя ошибка: "org.h2.mvstore.MVStoreException: The file is locked: C:/Users/komov/dev/java-shareit/db/shareit.mv.db [2.1.214/7]"
General error: "org.h2.mvstore.MVStoreException: The file is locked: C:/Users/komov/dev/java-shareit/db/shareit.mv.db [2.1.214/7]" [50000-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:554)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	... 52 more
Caused by: org.h2.mvstore.MVStoreException: The file is locked: C:/Users/komov/dev/java-shareit/db/shareit.mv.db [2.1.214/7]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:1004)
	at org.h2.mvstore.FileStore.open(FileStore.java:178)
	at org.h2.mvstore.FileStore.open(FileStore.java:128)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:452)
	... 46 more
2023-01-16 09:48:21 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_, user0_.email as email7_0_, user0_.name as name5_0_ from base_model user0_ where user0_.dtype='User' [42102-214]
2023-01-16 09:48:21 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_, user0_.email as email7_0_, user0_.name as name5_0_ from base_model user0_ where user0_.dtype='User' [42102-214]
2023-01-16 09:48:21 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_, user0_.email as email7_0_, user0_.name as name5_0_ from base_model user0_ where user0_.dtype='User' [42102-214]
2023-01-16 09:48:21 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_, user0_.email as email7_0_, user0_.name as name5_0_ from base_model user0_ where user0_.dtype='User' [42102-214]
2023-01-16 09:48:21 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_0_, user0_.email as email7_0_0_, user0_.name as name5_0_0_ from base_model user0_ where user0_.id=? and user0_.dtype='User' [42102-214]
2023-01-16 09:48:21 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_, user0_.email as email7_0_, user0_.name as name5_0_ from base_model user0_ where user0_.dtype='User' [42102-214]
2023-01-16 09:48:21 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_, user0_.email as email7_0_, user0_.name as name5_0_ from base_model user0_ where user0_.dtype='User' [42102-214]
2023-01-16 09:48:22 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_0_, user0_.email as email7_0_0_, user0_.name as name5_0_0_ from base_model user0_ where user0_.id=? and user0_.dtype='User' [42102-214]
2023-01-16 09:48:22 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_0_, user0_.email as email7_0_0_, user0_.name as name5_0_0_ from base_model user0_ where user0_.id=? and user0_.dtype='User' [42102-214]
2023-01-16 09:48:22 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_0_, user0_.email as email7_0_0_, user0_.name as name5_0_0_ from base_model user0_ where user0_.id=? and user0_.dtype='User' [42102-214]
2023-01-16 09:48:22 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_, user0_.email as email7_0_, user0_.name as name5_0_ from base_model user0_ where user0_.dtype='User' [42102-214]
2023-01-16 09:48:22 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_, user0_.email as email7_0_, user0_.name as name5_0_ from base_model user0_ where user0_.dtype='User' [42102-214]
2023-01-16 09:48:22 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_0_, user0_.email as email7_0_0_, user0_.name as name5_0_0_ from base_model user0_ where user0_.id=? and user0_.dtype='User' [42102-214]
2023-01-16 09:48:22 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_0_, user0_.email as email7_0_0_, user0_.name as name5_0_0_ from base_model user0_ where user0_.id=? and user0_.dtype='User' [42102-214]
2023-01-16 09:48:22 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select item0_.id as id2_0_0_, item0_.available as availabl3_0_0_, item0_.description as descript4_0_0_, item0_.name as name5_0_0_, item0_.owner_id as owner_id8_0_0_, item0_.request as request6_0_0_, user1_.id as id2_0_1_, user1_.email as email7_0_1_, user1_.name as name5_0_1_ from base_model item0_ left outer join base_model user1_ on item0_.owner_id=user1_.id where item0_.id=? and item0_.dtype='Item' [42102-214]
2023-01-16 09:48:22 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select item0_.id as id2_0_0_, item0_.available as availabl3_0_0_, item0_.description as descript4_0_0_, item0_.name as name5_0_0_, item0_.owner_id as owner_id8_0_0_, item0_.request as request6_0_0_, user1_.id as id2_0_1_, user1_.email as email7_0_1_, user1_.name as name5_0_1_ from base_model item0_ left outer join base_model user1_ on item0_.owner_id=user1_.id where item0_.id=? and item0_.dtype='Item' [42102-214]
2023-01-16 09:48:22 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select item0_.id as id2_0_0_, item0_.available as availabl3_0_0_, item0_.description as descript4_0_0_, item0_.name as name5_0_0_, item0_.owner_id as owner_id8_0_0_, item0_.request as request6_0_0_, user1_.id as id2_0_1_, user1_.email as email7_0_1_, user1_.name as name5_0_1_ from base_model item0_ left outer join base_model user1_ on item0_.owner_id=user1_.id where item0_.id=? and item0_.dtype='Item' [42102-214]
2023-01-16 09:48:23 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select item0_.id as id2_0_0_, item0_.available as availabl3_0_0_, item0_.description as descript4_0_0_, item0_.name as name5_0_0_, item0_.owner_id as owner_id8_0_0_, item0_.request as request6_0_0_, user1_.id as id2_0_1_, user1_.email as email7_0_1_, user1_.name as name5_0_1_ from base_model item0_ left outer join base_model user1_ on item0_.owner_id=user1_.id where item0_.id=? and item0_.dtype='Item' [42102-214]
2023-01-16 09:48:23 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select item0_.id as id2_0_0_, item0_.available as availabl3_0_0_, item0_.description as descript4_0_0_, item0_.name as name5_0_0_, item0_.owner_id as owner_id8_0_0_, item0_.request as request6_0_0_, user1_.id as id2_0_1_, user1_.email as email7_0_1_, user1_.name as name5_0_1_ from base_model item0_ left outer join base_model user1_ on item0_.owner_id=user1_.id where item0_.id=? and item0_.dtype='Item' [42102-214]
2023-01-16 09:48:23 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select item0_.id as id2_0_0_, item0_.available as availabl3_0_0_, item0_.description as descript4_0_0_, item0_.name as name5_0_0_, item0_.owner_id as owner_id8_0_0_, item0_.request as request6_0_0_, user1_.id as id2_0_1_, user1_.email as email7_0_1_, user1_.name as name5_0_1_ from base_model item0_ left outer join base_model user1_ on item0_.owner_id=user1_.id where item0_.id=? and item0_.dtype='Item' [42102-214]
2023-01-16 09:48:23 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_0_, user0_.email as email7_0_0_, user0_.name as name5_0_0_ from base_model user0_ where user0_.id=? and user0_.dtype='User' [42102-214]
2023-01-16 09:48:23 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select item0_.id as id2_0_, item0_.available as availabl3_0_, item0_.description as descript4_0_, item0_.name as name5_0_, item0_.owner_id as owner_id8_0_, item0_.request as request6_0_ from base_model item0_ where item0_.dtype='Item' [42102-214]
2023-01-16 09:48:23 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select item0_.id as id2_0_, item0_.available as availabl3_0_, item0_.description as descript4_0_, item0_.name as name5_0_, item0_.owner_id as owner_id8_0_, item0_.request as request6_0_ from base_model item0_ where item0_.dtype='Item' [42102-214]
2023-01-16 09:48:23 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select item0_.id as id2_0_, item0_.available as availabl3_0_, item0_.description as descript4_0_, item0_.name as name5_0_, item0_.owner_id as owner_id8_0_, item0_.request as request6_0_ from base_model item0_ where item0_.dtype='Item' [42102-214]
2023-01-16 09:48:23 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select item0_.id as id2_0_0_, item0_.available as availabl3_0_0_, item0_.description as descript4_0_0_, item0_.name as name5_0_0_, item0_.owner_id as owner_id8_0_0_, item0_.request as request6_0_0_, user1_.id as id2_0_1_, user1_.email as email7_0_1_, user1_.name as name5_0_1_ from base_model item0_ left outer join base_model user1_ on item0_.owner_id=user1_.id where item0_.id=? and item0_.dtype='Item' [42102-214]
2023-01-16 09:48:23 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select item0_.id as id2_0_, item0_.available as availabl3_0_, item0_.description as descript4_0_, item0_.name as name5_0_, item0_.owner_id as owner_id8_0_, item0_.request as request6_0_ from base_model item0_ where item0_.dtype='Item' [42102-214]
2023-01-16 09:48:23 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select item0_.id as id2_0_, item0_.available as availabl3_0_, item0_.description as descript4_0_, item0_.name as name5_0_, item0_.owner_id as owner_id8_0_, item0_.request as request6_0_ from base_model item0_ where item0_.dtype='Item' [42102-214]
2023-01-16 09:48:23 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select item0_.id as id2_0_0_, item0_.available as availabl3_0_0_, item0_.description as descript4_0_0_, item0_.name as name5_0_0_, item0_.owner_id as owner_id8_0_0_, item0_.request as request6_0_0_, user1_.id as id2_0_1_, user1_.email as email7_0_1_, user1_.name as name5_0_1_ from base_model item0_ left outer join base_model user1_ on item0_.owner_id=user1_.id where item0_.id=? and item0_.dtype='Item' [42102-214]
2023-01-16 09:48:23 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select item0_.id as id2_0_, item0_.available as availabl3_0_, item0_.description as descript4_0_, item0_.name as name5_0_, item0_.owner_id as owner_id8_0_, item0_.request as request6_0_ from base_model item0_ where item0_.dtype='Item' [42102-214]
2023-01-16 09:49:08 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_, user0_.email as email7_0_, user0_.name as name5_0_ from base_model user0_ where user0_.dtype='User' [42102-214]
2023-01-16 09:49:08 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_, user0_.email as email7_0_, user0_.name as name5_0_ from base_model user0_ where user0_.dtype='User' [42102-214]
2023-01-16 09:49:08 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_, user0_.email as email7_0_, user0_.name as name5_0_ from base_model user0_ where user0_.dtype='User' [42102-214]
2023-01-16 09:49:08 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_, user0_.email as email7_0_, user0_.name as name5_0_ from base_model user0_ where user0_.dtype='User' [42102-214]
2023-01-16 09:49:08 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_0_, user0_.email as email7_0_0_, user0_.name as name5_0_0_ from base_model user0_ where user0_.id=? and user0_.dtype='User' [42102-214]
2023-01-16 09:49:08 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_, user0_.email as email7_0_, user0_.name as name5_0_ from base_model user0_ where user0_.dtype='User' [42102-214]
2023-01-16 09:49:08 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_, user0_.email as email7_0_, user0_.name as name5_0_ from base_model user0_ where user0_.dtype='User' [42102-214]
2023-01-16 09:49:08 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_0_, user0_.email as email7_0_0_, user0_.name as name5_0_0_ from base_model user0_ where user0_.id=? and user0_.dtype='User' [42102-214]
2023-01-16 09:49:09 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_0_, user0_.email as email7_0_0_, user0_.name as name5_0_0_ from base_model user0_ where user0_.id=? and user0_.dtype='User' [42102-214]
2023-01-16 09:49:09 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_0_, user0_.email as email7_0_0_, user0_.name as name5_0_0_ from base_model user0_ where user0_.id=? and user0_.dtype='User' [42102-214]
2023-01-16 09:49:09 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_, user0_.email as email7_0_, user0_.name as name5_0_ from base_model user0_ where user0_.dtype='User' [42102-214]
2023-01-16 09:49:09 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_, user0_.email as email7_0_, user0_.name as name5_0_ from base_model user0_ where user0_.dtype='User' [42102-214]
2023-01-16 09:49:09 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_0_, user0_.email as email7_0_0_, user0_.name as name5_0_0_ from base_model user0_ where user0_.id=? and user0_.dtype='User' [42102-214]
2023-01-16 09:49:09 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_0_, user0_.email as email7_0_0_, user0_.name as name5_0_0_ from base_model user0_ where user0_.id=? and user0_.dtype='User' [42102-214]
2023-01-16 09:49:09 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select item0_.id as id2_0_0_, item0_.available as availabl3_0_0_, item0_.description as descript4_0_0_, item0_.name as name5_0_0_, item0_.owner_id as owner_id8_0_0_, item0_.request as request6_0_0_, user1_.id as id2_0_1_, user1_.email as email7_0_1_, user1_.name as name5_0_1_ from base_model item0_ left outer join base_model user1_ on item0_.owner_id=user1_.id where item0_.id=? and item0_.dtype='Item' [42102-214]
2023-01-16 09:49:09 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select item0_.id as id2_0_0_, item0_.available as availabl3_0_0_, item0_.description as descript4_0_0_, item0_.name as name5_0_0_, item0_.owner_id as owner_id8_0_0_, item0_.request as request6_0_0_, user1_.id as id2_0_1_, user1_.email as email7_0_1_, user1_.name as name5_0_1_ from base_model item0_ left outer join base_model user1_ on item0_.owner_id=user1_.id where item0_.id=? and item0_.dtype='Item' [42102-214]
2023-01-16 09:49:09 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select item0_.id as id2_0_0_, item0_.available as availabl3_0_0_, item0_.description as descript4_0_0_, item0_.name as name5_0_0_, item0_.owner_id as owner_id8_0_0_, item0_.request as request6_0_0_, user1_.id as id2_0_1_, user1_.email as email7_0_1_, user1_.name as name5_0_1_ from base_model item0_ left outer join base_model user1_ on item0_.owner_id=user1_.id where item0_.id=? and item0_.dtype='Item' [42102-214]
2023-01-16 09:49:09 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select item0_.id as id2_0_0_, item0_.available as availabl3_0_0_, item0_.description as descript4_0_0_, item0_.name as name5_0_0_, item0_.owner_id as owner_id8_0_0_, item0_.request as request6_0_0_, user1_.id as id2_0_1_, user1_.email as email7_0_1_, user1_.name as name5_0_1_ from base_model item0_ left outer join base_model user1_ on item0_.owner_id=user1_.id where item0_.id=? and item0_.dtype='Item' [42102-214]
2023-01-16 09:49:09 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select item0_.id as id2_0_0_, item0_.available as availabl3_0_0_, item0_.description as descript4_0_0_, item0_.name as name5_0_0_, item0_.owner_id as owner_id8_0_0_, item0_.request as request6_0_0_, user1_.id as id2_0_1_, user1_.email as email7_0_1_, user1_.name as name5_0_1_ from base_model item0_ left outer join base_model user1_ on item0_.owner_id=user1_.id where item0_.id=? and item0_.dtype='Item' [42102-214]
2023-01-16 09:49:09 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select item0_.id as id2_0_0_, item0_.available as availabl3_0_0_, item0_.description as descript4_0_0_, item0_.name as name5_0_0_, item0_.owner_id as owner_id8_0_0_, item0_.request as request6_0_0_, user1_.id as id2_0_1_, user1_.email as email7_0_1_, user1_.name as name5_0_1_ from base_model item0_ left outer join base_model user1_ on item0_.owner_id=user1_.id where item0_.id=? and item0_.dtype='Item' [42102-214]
2023-01-16 09:49:09 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_0_, user0_.email as email7_0_0_, user0_.name as name5_0_0_ from base_model user0_ where user0_.id=? and user0_.dtype='User' [42102-214]
2023-01-16 09:49:09 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select item0_.id as id2_0_, item0_.available as availabl3_0_, item0_.description as descript4_0_, item0_.name as name5_0_, item0_.owner_id as owner_id8_0_, item0_.request as request6_0_ from base_model item0_ where item0_.dtype='Item' [42102-214]
2023-01-16 09:49:09 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select item0_.id as id2_0_, item0_.available as availabl3_0_, item0_.description as descript4_0_, item0_.name as name5_0_, item0_.owner_id as owner_id8_0_, item0_.request as request6_0_ from base_model item0_ where item0_.dtype='Item' [42102-214]
2023-01-16 09:49:09 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select item0_.id as id2_0_, item0_.available as availabl3_0_, item0_.description as descript4_0_, item0_.name as name5_0_, item0_.owner_id as owner_id8_0_, item0_.request as request6_0_ from base_model item0_ where item0_.dtype='Item' [42102-214]
2023-01-16 09:49:09 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select item0_.id as id2_0_0_, item0_.available as availabl3_0_0_, item0_.description as descript4_0_0_, item0_.name as name5_0_0_, item0_.owner_id as owner_id8_0_0_, item0_.request as request6_0_0_, user1_.id as id2_0_1_, user1_.email as email7_0_1_, user1_.name as name5_0_1_ from base_model item0_ left outer join base_model user1_ on item0_.owner_id=user1_.id where item0_.id=? and item0_.dtype='Item' [42102-214]
2023-01-16 09:49:10 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select item0_.id as id2_0_, item0_.available as availabl3_0_, item0_.description as descript4_0_, item0_.name as name5_0_, item0_.owner_id as owner_id8_0_, item0_.request as request6_0_ from base_model item0_ where item0_.dtype='Item' [42102-214]
2023-01-16 09:49:10 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select item0_.id as id2_0_, item0_.available as availabl3_0_, item0_.description as descript4_0_, item0_.name as name5_0_, item0_.owner_id as owner_id8_0_, item0_.request as request6_0_ from base_model item0_ where item0_.dtype='Item' [42102-214]
2023-01-16 09:49:10 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select item0_.id as id2_0_0_, item0_.available as availabl3_0_0_, item0_.description as descript4_0_0_, item0_.name as name5_0_0_, item0_.owner_id as owner_id8_0_0_, item0_.request as request6_0_0_, user1_.id as id2_0_1_, user1_.email as email7_0_1_, user1_.name as name5_0_1_ from base_model item0_ left outer join base_model user1_ on item0_.owner_id=user1_.id where item0_.id=? and item0_.dtype='Item' [42102-214]
2023-01-16 09:49:10 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select item0_.id as id2_0_, item0_.available as availabl3_0_, item0_.description as descript4_0_, item0_.name as name5_0_, item0_.owner_id as owner_id8_0_, item0_.request as request6_0_ from base_model item0_ where item0_.dtype='Item' [42102-214]
2023-01-16 09:52:23 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_, user0_.email as email7_0_, user0_.name as name5_0_ from base_model user0_ where user0_.dtype='User' [42102-214]
2023-01-16 09:52:23 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_, user0_.email as email7_0_, user0_.name as name5_0_ from base_model user0_ where user0_.dtype='User' [42102-214]
2023-01-16 09:52:23 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_, user0_.email as email7_0_, user0_.name as name5_0_ from base_model user0_ where user0_.dtype='User' [42102-214]
2023-01-16 09:52:23 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_, user0_.email as email7_0_, user0_.name as name5_0_ from base_model user0_ where user0_.dtype='User' [42102-214]
2023-01-16 09:52:23 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_0_, user0_.email as email7_0_0_, user0_.name as name5_0_0_ from base_model user0_ where user0_.id=? and user0_.dtype='User' [42102-214]
2023-01-16 09:52:23 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_, user0_.email as email7_0_, user0_.name as name5_0_ from base_model user0_ where user0_.dtype='User' [42102-214]
2023-01-16 09:52:23 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_, user0_.email as email7_0_, user0_.name as name5_0_ from base_model user0_ where user0_.dtype='User' [42102-214]
2023-01-16 09:52:23 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_0_, user0_.email as email7_0_0_, user0_.name as name5_0_0_ from base_model user0_ where user0_.id=? and user0_.dtype='User' [42102-214]
2023-01-16 09:52:23 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_0_, user0_.email as email7_0_0_, user0_.name as name5_0_0_ from base_model user0_ where user0_.id=? and user0_.dtype='User' [42102-214]
2023-01-16 09:52:23 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_0_, user0_.email as email7_0_0_, user0_.name as name5_0_0_ from base_model user0_ where user0_.id=? and user0_.dtype='User' [42102-214]
2023-01-16 09:52:23 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_, user0_.email as email7_0_, user0_.name as name5_0_ from base_model user0_ where user0_.dtype='User' [42102-214]
2023-01-16 09:52:23 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_, user0_.email as email7_0_, user0_.name as name5_0_ from base_model user0_ where user0_.dtype='User' [42102-214]
2023-01-16 09:52:24 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_0_, user0_.email as email7_0_0_, user0_.name as name5_0_0_ from base_model user0_ where user0_.id=? and user0_.dtype='User' [42102-214]
2023-01-16 09:52:24 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_0_, user0_.email as email7_0_0_, user0_.name as name5_0_0_ from base_model user0_ where user0_.id=? and user0_.dtype='User' [42102-214]
2023-01-16 09:52:24 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select item0_.id as id2_0_0_, item0_.available as availabl3_0_0_, item0_.description as descript4_0_0_, item0_.name as name5_0_0_, item0_.owner_id as owner_id8_0_0_, item0_.request as request6_0_0_, user1_.id as id2_0_1_, user1_.email as email7_0_1_, user1_.name as name5_0_1_ from base_model item0_ left outer join base_model user1_ on item0_.owner_id=user1_.id where item0_.id=? and item0_.dtype='Item' [42102-214]
2023-01-16 09:52:24 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select item0_.id as id2_0_0_, item0_.available as availabl3_0_0_, item0_.description as descript4_0_0_, item0_.name as name5_0_0_, item0_.owner_id as owner_id8_0_0_, item0_.request as request6_0_0_, user1_.id as id2_0_1_, user1_.email as email7_0_1_, user1_.name as name5_0_1_ from base_model item0_ left outer join base_model user1_ on item0_.owner_id=user1_.id where item0_.id=? and item0_.dtype='Item' [42102-214]
2023-01-16 09:52:24 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select item0_.id as id2_0_0_, item0_.available as availabl3_0_0_, item0_.description as descript4_0_0_, item0_.name as name5_0_0_, item0_.owner_id as owner_id8_0_0_, item0_.request as request6_0_0_, user1_.id as id2_0_1_, user1_.email as email7_0_1_, user1_.name as name5_0_1_ from base_model item0_ left outer join base_model user1_ on item0_.owner_id=user1_.id where item0_.id=? and item0_.dtype='Item' [42102-214]
2023-01-16 09:52:24 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select item0_.id as id2_0_0_, item0_.available as availabl3_0_0_, item0_.description as descript4_0_0_, item0_.name as name5_0_0_, item0_.owner_id as owner_id8_0_0_, item0_.request as request6_0_0_, user1_.id as id2_0_1_, user1_.email as email7_0_1_, user1_.name as name5_0_1_ from base_model item0_ left outer join base_model user1_ on item0_.owner_id=user1_.id where item0_.id=? and item0_.dtype='Item' [42102-214]
2023-01-16 09:52:24 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select item0_.id as id2_0_0_, item0_.available as availabl3_0_0_, item0_.description as descript4_0_0_, item0_.name as name5_0_0_, item0_.owner_id as owner_id8_0_0_, item0_.request as request6_0_0_, user1_.id as id2_0_1_, user1_.email as email7_0_1_, user1_.name as name5_0_1_ from base_model item0_ left outer join base_model user1_ on item0_.owner_id=user1_.id where item0_.id=? and item0_.dtype='Item' [42102-214]
2023-01-16 09:52:24 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select item0_.id as id2_0_0_, item0_.available as availabl3_0_0_, item0_.description as descript4_0_0_, item0_.name as name5_0_0_, item0_.owner_id as owner_id8_0_0_, item0_.request as request6_0_0_, user1_.id as id2_0_1_, user1_.email as email7_0_1_, user1_.name as name5_0_1_ from base_model item0_ left outer join base_model user1_ on item0_.owner_id=user1_.id where item0_.id=? and item0_.dtype='Item' [42102-214]
2023-01-16 09:52:24 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_0_, user0_.email as email7_0_0_, user0_.name as name5_0_0_ from base_model user0_ where user0_.id=? and user0_.dtype='User' [42102-214]
2023-01-16 09:52:24 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select item0_.id as id2_0_, item0_.available as availabl3_0_, item0_.description as descript4_0_, item0_.name as name5_0_, item0_.owner_id as owner_id8_0_, item0_.request as request6_0_ from base_model item0_ where item0_.dtype='Item' [42102-214]
2023-01-16 09:52:24 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select item0_.id as id2_0_, item0_.available as availabl3_0_, item0_.description as descript4_0_, item0_.name as name5_0_, item0_.owner_id as owner_id8_0_, item0_.request as request6_0_ from base_model item0_ where item0_.dtype='Item' [42102-214]
2023-01-16 09:52:24 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select item0_.id as id2_0_, item0_.available as availabl3_0_, item0_.description as descript4_0_, item0_.name as name5_0_, item0_.owner_id as owner_id8_0_, item0_.request as request6_0_ from base_model item0_ where item0_.dtype='Item' [42102-214]
2023-01-16 09:52:24 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select item0_.id as id2_0_0_, item0_.available as availabl3_0_0_, item0_.description as descript4_0_0_, item0_.name as name5_0_0_, item0_.owner_id as owner_id8_0_0_, item0_.request as request6_0_0_, user1_.id as id2_0_1_, user1_.email as email7_0_1_, user1_.name as name5_0_1_ from base_model item0_ left outer join base_model user1_ on item0_.owner_id=user1_.id where item0_.id=? and item0_.dtype='Item' [42102-214]
2023-01-16 09:52:24 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select item0_.id as id2_0_, item0_.available as availabl3_0_, item0_.description as descript4_0_, item0_.name as name5_0_, item0_.owner_id as owner_id8_0_, item0_.request as request6_0_ from base_model item0_ where item0_.dtype='Item' [42102-214]
2023-01-16 09:52:24 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select item0_.id as id2_0_, item0_.available as availabl3_0_, item0_.description as descript4_0_, item0_.name as name5_0_, item0_.owner_id as owner_id8_0_, item0_.request as request6_0_ from base_model item0_ where item0_.dtype='Item' [42102-214]
2023-01-16 09:52:24 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select item0_.id as id2_0_0_, item0_.available as availabl3_0_0_, item0_.description as descript4_0_0_, item0_.name as name5_0_0_, item0_.owner_id as owner_id8_0_0_, item0_.request as request6_0_0_, user1_.id as id2_0_1_, user1_.email as email7_0_1_, user1_.name as name5_0_1_ from base_model item0_ left outer join base_model user1_ on item0_.owner_id=user1_.id where item0_.id=? and item0_.dtype='Item' [42102-214]
2023-01-16 09:52:24 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select item0_.id as id2_0_, item0_.available as availabl3_0_, item0_.description as descript4_0_, item0_.name as name5_0_, item0_.owner_id as owner_id8_0_, item0_.request as request6_0_ from base_model item0_ where item0_.dtype='Item' [42102-214]
2023-01-16 09:53:23 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_, user0_.email as email7_0_, user0_.name as name5_0_ from base_model user0_ where user0_.dtype='User' [42102-214]
2023-01-16 09:53:24 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_, user0_.email as email7_0_, user0_.name as name5_0_ from base_model user0_ where user0_.dtype='User' [42102-214]
2023-01-16 09:53:24 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_, user0_.email as email7_0_, user0_.name as name5_0_ from base_model user0_ where user0_.dtype='User' [42102-214]
2023-01-16 09:53:24 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_, user0_.email as email7_0_, user0_.name as name5_0_ from base_model user0_ where user0_.dtype='User' [42102-214]
2023-01-16 09:53:24 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_0_, user0_.email as email7_0_0_, user0_.name as name5_0_0_ from base_model user0_ where user0_.id=? and user0_.dtype='User' [42102-214]
2023-01-16 09:53:24 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_, user0_.email as email7_0_, user0_.name as name5_0_ from base_model user0_ where user0_.dtype='User' [42102-214]
2023-01-16 09:53:24 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_, user0_.email as email7_0_, user0_.name as name5_0_ from base_model user0_ where user0_.dtype='User' [42102-214]
2023-01-16 09:53:24 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_0_, user0_.email as email7_0_0_, user0_.name as name5_0_0_ from base_model user0_ where user0_.id=? and user0_.dtype='User' [42102-214]
2023-01-16 09:53:24 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_0_, user0_.email as email7_0_0_, user0_.name as name5_0_0_ from base_model user0_ where user0_.id=? and user0_.dtype='User' [42102-214]
2023-01-16 09:53:24 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_0_, user0_.email as email7_0_0_, user0_.name as name5_0_0_ from base_model user0_ where user0_.id=? and user0_.dtype='User' [42102-214]
2023-01-16 09:53:24 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_, user0_.email as email7_0_, user0_.name as name5_0_ from base_model user0_ where user0_.dtype='User' [42102-214]
2023-01-16 09:53:24 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_, user0_.email as email7_0_, user0_.name as name5_0_ from base_model user0_ where user0_.dtype='User' [42102-214]
2023-01-16 09:53:24 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_0_, user0_.email as email7_0_0_, user0_.name as name5_0_0_ from base_model user0_ where user0_.id=? and user0_.dtype='User' [42102-214]
2023-01-16 09:53:24 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_0_, user0_.email as email7_0_0_, user0_.name as name5_0_0_ from base_model user0_ where user0_.id=? and user0_.dtype='User' [42102-214]
2023-01-16 09:53:24 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select item0_.id as id2_0_0_, item0_.available as availabl3_0_0_, item0_.description as descript4_0_0_, item0_.name as name5_0_0_, item0_.owner_id as owner_id8_0_0_, item0_.request as request6_0_0_, user1_.id as id2_0_1_, user1_.email as email7_0_1_, user1_.name as name5_0_1_ from base_model item0_ left outer join base_model user1_ on item0_.owner_id=user1_.id where item0_.id=? and item0_.dtype='Item' [42102-214]
2023-01-16 09:53:24 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select item0_.id as id2_0_0_, item0_.available as availabl3_0_0_, item0_.description as descript4_0_0_, item0_.name as name5_0_0_, item0_.owner_id as owner_id8_0_0_, item0_.request as request6_0_0_, user1_.id as id2_0_1_, user1_.email as email7_0_1_, user1_.name as name5_0_1_ from base_model item0_ left outer join base_model user1_ on item0_.owner_id=user1_.id where item0_.id=? and item0_.dtype='Item' [42102-214]
2023-01-16 09:53:24 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select item0_.id as id2_0_0_, item0_.available as availabl3_0_0_, item0_.description as descript4_0_0_, item0_.name as name5_0_0_, item0_.owner_id as owner_id8_0_0_, item0_.request as request6_0_0_, user1_.id as id2_0_1_, user1_.email as email7_0_1_, user1_.name as name5_0_1_ from base_model item0_ left outer join base_model user1_ on item0_.owner_id=user1_.id where item0_.id=? and item0_.dtype='Item' [42102-214]
2023-01-16 09:53:24 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select item0_.id as id2_0_0_, item0_.available as availabl3_0_0_, item0_.description as descript4_0_0_, item0_.name as name5_0_0_, item0_.owner_id as owner_id8_0_0_, item0_.request as request6_0_0_, user1_.id as id2_0_1_, user1_.email as email7_0_1_, user1_.name as name5_0_1_ from base_model item0_ left outer join base_model user1_ on item0_.owner_id=user1_.id where item0_.id=? and item0_.dtype='Item' [42102-214]
2023-01-16 09:53:24 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select item0_.id as id2_0_0_, item0_.available as availabl3_0_0_, item0_.description as descript4_0_0_, item0_.name as name5_0_0_, item0_.owner_id as owner_id8_0_0_, item0_.request as request6_0_0_, user1_.id as id2_0_1_, user1_.email as email7_0_1_, user1_.name as name5_0_1_ from base_model item0_ left outer join base_model user1_ on item0_.owner_id=user1_.id where item0_.id=? and item0_.dtype='Item' [42102-214]
2023-01-16 09:53:24 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select item0_.id as id2_0_0_, item0_.available as availabl3_0_0_, item0_.description as descript4_0_0_, item0_.name as name5_0_0_, item0_.owner_id as owner_id8_0_0_, item0_.request as request6_0_0_, user1_.id as id2_0_1_, user1_.email as email7_0_1_, user1_.name as name5_0_1_ from base_model item0_ left outer join base_model user1_ on item0_.owner_id=user1_.id where item0_.id=? and item0_.dtype='Item' [42102-214]
2023-01-16 09:53:24 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_0_, user0_.email as email7_0_0_, user0_.name as name5_0_0_ from base_model user0_ where user0_.id=? and user0_.dtype='User' [42102-214]
2023-01-16 09:53:24 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select item0_.id as id2_0_, item0_.available as availabl3_0_, item0_.description as descript4_0_, item0_.name as name5_0_, item0_.owner_id as owner_id8_0_, item0_.request as request6_0_ from base_model item0_ where item0_.dtype='Item' [42102-214]
2023-01-16 09:53:24 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select item0_.id as id2_0_, item0_.available as availabl3_0_, item0_.description as descript4_0_, item0_.name as name5_0_, item0_.owner_id as owner_id8_0_, item0_.request as request6_0_ from base_model item0_ where item0_.dtype='Item' [42102-214]
2023-01-16 09:53:25 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select item0_.id as id2_0_, item0_.available as availabl3_0_, item0_.description as descript4_0_, item0_.name as name5_0_, item0_.owner_id as owner_id8_0_, item0_.request as request6_0_ from base_model item0_ where item0_.dtype='Item' [42102-214]
2023-01-16 09:53:25 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select item0_.id as id2_0_0_, item0_.available as availabl3_0_0_, item0_.description as descript4_0_0_, item0_.name as name5_0_0_, item0_.owner_id as owner_id8_0_0_, item0_.request as request6_0_0_, user1_.id as id2_0_1_, user1_.email as email7_0_1_, user1_.name as name5_0_1_ from base_model item0_ left outer join base_model user1_ on item0_.owner_id=user1_.id where item0_.id=? and item0_.dtype='Item' [42102-214]
2023-01-16 09:53:25 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select item0_.id as id2_0_, item0_.available as availabl3_0_, item0_.description as descript4_0_, item0_.name as name5_0_, item0_.owner_id as owner_id8_0_, item0_.request as request6_0_ from base_model item0_ where item0_.dtype='Item' [42102-214]
2023-01-16 09:53:25 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select item0_.id as id2_0_, item0_.available as availabl3_0_, item0_.description as descript4_0_, item0_.name as name5_0_, item0_.owner_id as owner_id8_0_, item0_.request as request6_0_ from base_model item0_ where item0_.dtype='Item' [42102-214]
2023-01-16 09:53:25 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select item0_.id as id2_0_0_, item0_.available as availabl3_0_0_, item0_.description as descript4_0_0_, item0_.name as name5_0_0_, item0_.owner_id as owner_id8_0_0_, item0_.request as request6_0_0_, user1_.id as id2_0_1_, user1_.email as email7_0_1_, user1_.name as name5_0_1_ from base_model item0_ left outer join base_model user1_ on item0_.owner_id=user1_.id where item0_.id=? and item0_.dtype='Item' [42102-214]
2023-01-16 09:53:25 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select item0_.id as id2_0_, item0_.available as availabl3_0_, item0_.description as descript4_0_, item0_.name as name5_0_, item0_.owner_id as owner_id8_0_, item0_.request as request6_0_ from base_model item0_ where item0_.dtype='Item' [42102-214]
2023-01-16 06:53:44 jdbc[3]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1573)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:470)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:360)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:712)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:587)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:828)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:705)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:704)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
2023-01-16 09:54:36 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_, user0_.email as email7_0_, user0_.name as name5_0_ from base_model user0_ where user0_.dtype='User' [42102-214]
2023-01-16 09:54:36 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_, user0_.email as email7_0_, user0_.name as name5_0_ from base_model user0_ where user0_.dtype='User' [42102-214]
2023-01-16 09:54:36 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_, user0_.email as email7_0_, user0_.name as name5_0_ from base_model user0_ where user0_.dtype='User' [42102-214]
2023-01-16 09:54:36 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_, user0_.email as email7_0_, user0_.name as name5_0_ from base_model user0_ where user0_.dtype='User' [42102-214]
2023-01-16 09:54:36 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_0_, user0_.email as email7_0_0_, user0_.name as name5_0_0_ from base_model user0_ where user0_.id=? and user0_.dtype='User' [42102-214]
2023-01-16 09:54:36 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_, user0_.email as email7_0_, user0_.name as name5_0_ from base_model user0_ where user0_.dtype='User' [42102-214]
2023-01-16 09:54:36 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_, user0_.email as email7_0_, user0_.name as name5_0_ from base_model user0_ where user0_.dtype='User' [42102-214]
2023-01-16 09:54:36 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_0_, user0_.email as email7_0_0_, user0_.name as name5_0_0_ from base_model user0_ where user0_.id=? and user0_.dtype='User' [42102-214]
2023-01-16 09:54:36 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_0_, user0_.email as email7_0_0_, user0_.name as name5_0_0_ from base_model user0_ where user0_.id=? and user0_.dtype='User' [42102-214]
2023-01-16 09:54:36 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_0_, user0_.email as email7_0_0_, user0_.name as name5_0_0_ from base_model user0_ where user0_.id=? and user0_.dtype='User' [42102-214]
2023-01-16 09:54:36 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_, user0_.email as email7_0_, user0_.name as name5_0_ from base_model user0_ where user0_.dtype='User' [42102-214]
2023-01-16 09:54:37 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_, user0_.email as email7_0_, user0_.name as name5_0_ from base_model user0_ where user0_.dtype='User' [42102-214]
2023-01-16 09:54:37 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_0_, user0_.email as email7_0_0_, user0_.name as name5_0_0_ from base_model user0_ where user0_.id=? and user0_.dtype='User' [42102-214]
2023-01-16 09:54:37 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_0_, user0_.email as email7_0_0_, user0_.name as name5_0_0_ from base_model user0_ where user0_.id=? and user0_.dtype='User' [42102-214]
2023-01-16 09:54:37 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select item0_.id as id2_0_0_, item0_.available as availabl3_0_0_, item0_.description as descript4_0_0_, item0_.name as name5_0_0_, item0_.owner_id as owner_id8_0_0_, item0_.request as request6_0_0_, user1_.id as id2_0_1_, user1_.email as email7_0_1_, user1_.name as name5_0_1_ from base_model item0_ left outer join base_model user1_ on item0_.owner_id=user1_.id where item0_.id=? and item0_.dtype='Item' [42102-214]
2023-01-16 09:54:37 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select item0_.id as id2_0_0_, item0_.available as availabl3_0_0_, item0_.description as descript4_0_0_, item0_.name as name5_0_0_, item0_.owner_id as owner_id8_0_0_, item0_.request as request6_0_0_, user1_.id as id2_0_1_, user1_.email as email7_0_1_, user1_.name as name5_0_1_ from base_model item0_ left outer join base_model user1_ on item0_.owner_id=user1_.id where item0_.id=? and item0_.dtype='Item' [42102-214]
2023-01-16 09:54:37 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select item0_.id as id2_0_0_, item0_.available as availabl3_0_0_, item0_.description as descript4_0_0_, item0_.name as name5_0_0_, item0_.owner_id as owner_id8_0_0_, item0_.request as request6_0_0_, user1_.id as id2_0_1_, user1_.email as email7_0_1_, user1_.name as name5_0_1_ from base_model item0_ left outer join base_model user1_ on item0_.owner_id=user1_.id where item0_.id=? and item0_.dtype='Item' [42102-214]
2023-01-16 09:54:37 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select item0_.id as id2_0_0_, item0_.available as availabl3_0_0_, item0_.description as descript4_0_0_, item0_.name as name5_0_0_, item0_.owner_id as owner_id8_0_0_, item0_.request as request6_0_0_, user1_.id as id2_0_1_, user1_.email as email7_0_1_, user1_.name as name5_0_1_ from base_model item0_ left outer join base_model user1_ on item0_.owner_id=user1_.id where item0_.id=? and item0_.dtype='Item' [42102-214]
2023-01-16 09:54:37 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select item0_.id as id2_0_0_, item0_.available as availabl3_0_0_, item0_.description as descript4_0_0_, item0_.name as name5_0_0_, item0_.owner_id as owner_id8_0_0_, item0_.request as request6_0_0_, user1_.id as id2_0_1_, user1_.email as email7_0_1_, user1_.name as name5_0_1_ from base_model item0_ left outer join base_model user1_ on item0_.owner_id=user1_.id where item0_.id=? and item0_.dtype='Item' [42102-214]
2023-01-16 09:54:37 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select item0_.id as id2_0_0_, item0_.available as availabl3_0_0_, item0_.description as descript4_0_0_, item0_.name as name5_0_0_, item0_.owner_id as owner_id8_0_0_, item0_.request as request6_0_0_, user1_.id as id2_0_1_, user1_.email as email7_0_1_, user1_.name as name5_0_1_ from base_model item0_ left outer join base_model user1_ on item0_.owner_id=user1_.id where item0_.id=? and item0_.dtype='Item' [42102-214]
2023-01-16 09:54:37 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_0_, user0_.email as email7_0_0_, user0_.name as name5_0_0_ from base_model user0_ where user0_.id=? and user0_.dtype='User' [42102-214]
2023-01-16 09:54:37 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select item0_.id as id2_0_, item0_.available as availabl3_0_, item0_.description as descript4_0_, item0_.name as name5_0_, item0_.owner_id as owner_id8_0_, item0_.request as request6_0_ from base_model item0_ where item0_.dtype='Item' [42102-214]
2023-01-16 09:54:37 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select item0_.id as id2_0_, item0_.available as availabl3_0_, item0_.description as descript4_0_, item0_.name as name5_0_, item0_.owner_id as owner_id8_0_, item0_.request as request6_0_ from base_model item0_ where item0_.dtype='Item' [42102-214]
2023-01-16 09:54:37 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select item0_.id as id2_0_, item0_.available as availabl3_0_, item0_.description as descript4_0_, item0_.name as name5_0_, item0_.owner_id as owner_id8_0_, item0_.request as request6_0_ from base_model item0_ where item0_.dtype='Item' [42102-214]
2023-01-16 09:54:37 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select item0_.id as id2_0_0_, item0_.available as availabl3_0_0_, item0_.description as descript4_0_0_, item0_.name as name5_0_0_, item0_.owner_id as owner_id8_0_0_, item0_.request as request6_0_0_, user1_.id as id2_0_1_, user1_.email as email7_0_1_, user1_.name as name5_0_1_ from base_model item0_ left outer join base_model user1_ on item0_.owner_id=user1_.id where item0_.id=? and item0_.dtype='Item' [42102-214]
2023-01-16 09:54:37 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select item0_.id as id2_0_, item0_.available as availabl3_0_, item0_.description as descript4_0_, item0_.name as name5_0_, item0_.owner_id as owner_id8_0_, item0_.request as request6_0_ from base_model item0_ where item0_.dtype='Item' [42102-214]
2023-01-16 09:54:37 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select item0_.id as id2_0_, item0_.available as availabl3_0_, item0_.description as descript4_0_, item0_.name as name5_0_, item0_.owner_id as owner_id8_0_, item0_.request as request6_0_ from base_model item0_ where item0_.dtype='Item' [42102-214]
2023-01-16 09:54:37 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select item0_.id as id2_0_0_, item0_.available as availabl3_0_0_, item0_.description as descript4_0_0_, item0_.name as name5_0_0_, item0_.owner_id as owner_id8_0_0_, item0_.request as request6_0_0_, user1_.id as id2_0_1_, user1_.email as email7_0_1_, user1_.name as name5_0_1_ from base_model item0_ left outer join base_model user1_ on item0_.owner_id=user1_.id where item0_.id=? and item0_.dtype='Item' [42102-214]
2023-01-16 09:54:37 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select item0_.id as id2_0_, item0_.available as availabl3_0_, item0_.description as descript4_0_, item0_.name as name5_0_, item0_.owner_id as owner_id8_0_, item0_.request as request6_0_ from base_model item0_ where item0_.dtype='Item' [42102-214]
2023-01-16 09:55:07 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_, user0_.email as email7_0_, user0_.name as name5_0_ from base_model user0_ where user0_.dtype='User' [42102-214]
2023-01-16 09:56:11 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_, user0_.email as email7_0_, user0_.name as name5_0_ from base_model user0_ where user0_.dtype='User' [42102-214]
2023-01-16 09:57:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_, user0_.email as email7_0_, user0_.name as name5_0_ from base_model user0_ where user0_.dtype='User' [42102-214]
2023-01-16 09:59:29 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_, user0_.email as email7_0_, user0_.name as name5_0_ from base_model user0_ where user0_.dtype='User' [42102-214]
2023-01-16 09:59:45 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
insert into base_model (email, name, dtype) values (?, ?, 'User') [42102-214]
2023-01-16 09:59:45 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
insert into base_model (email, name, dtype) values (?, ?, 'User') [42102-214]
2023-01-16 09:59:45 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_0_, user0_.email as email7_0_0_, user0_.name as name5_0_0_ from base_model user0_ where user0_.id=? and user0_.dtype='User' [42102-214]
2023-01-16 09:59:45 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
insert into base_model (email, name, dtype) values (?, ?, 'User') [42102-214]
2023-01-16 09:59:45 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_0_, user0_.email as email7_0_0_, user0_.name as name5_0_0_ from base_model user0_ where user0_.id=? and user0_.dtype='User' [42102-214]
2023-01-16 09:59:45 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_0_, user0_.email as email7_0_0_, user0_.name as name5_0_0_ from base_model user0_ where user0_.id=? and user0_.dtype='User' [42102-214]
2023-01-16 09:59:45 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_0_, user0_.email as email7_0_0_, user0_.name as name5_0_0_ from base_model user0_ where user0_.id=? and user0_.dtype='User' [42102-214]
2023-01-16 09:59:45 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_0_, user0_.email as email7_0_0_, user0_.name as name5_0_0_ from base_model user0_ where user0_.id=? and user0_.dtype='User' [42102-214]
2023-01-16 09:59:45 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_0_, user0_.email as email7_0_0_, user0_.name as name5_0_0_ from base_model user0_ where user0_.id=? and user0_.dtype='User' [42102-214]
2023-01-16 09:59:45 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_0_, user0_.email as email7_0_0_, user0_.name as name5_0_0_ from base_model user0_ where user0_.id=? and user0_.dtype='User' [42102-214]
2023-01-16 09:59:45 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
insert into base_model (email, name, dtype) values (?, ?, 'User') [42102-214]
2023-01-16 09:59:45 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_, user0_.email as email7_0_, user0_.name as name5_0_ from base_model user0_ where user0_.dtype='User' [42102-214]
2023-01-16 09:59:46 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_0_, user0_.email as email7_0_0_, user0_.name as name5_0_0_ from base_model user0_ where user0_.id=? and user0_.dtype='User' [42102-214]
2023-01-16 09:59:46 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_0_, user0_.email as email7_0_0_, user0_.name as name5_0_0_ from base_model user0_ where user0_.id=? and user0_.dtype='User' [42102-214]
2023-01-16 09:59:46 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select item0_.id as id2_0_0_, item0_.available as availabl3_0_0_, item0_.description as descript4_0_0_, item0_.name as name5_0_0_, item0_.owner_id as owner_id8_0_0_, item0_.request as request6_0_0_, user1_.id as id2_0_1_, user1_.email as email7_0_1_, user1_.name as name5_0_1_ from base_model item0_ left outer join base_model user1_ on item0_.owner_id=user1_.id where item0_.id=? and item0_.dtype='Item' [42102-214]
2023-01-16 09:59:46 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select item0_.id as id2_0_0_, item0_.available as availabl3_0_0_, item0_.description as descript4_0_0_, item0_.name as name5_0_0_, item0_.owner_id as owner_id8_0_0_, item0_.request as request6_0_0_, user1_.id as id2_0_1_, user1_.email as email7_0_1_, user1_.name as name5_0_1_ from base_model item0_ left outer join base_model user1_ on item0_.owner_id=user1_.id where item0_.id=? and item0_.dtype='Item' [42102-214]
2023-01-16 09:59:46 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select item0_.id as id2_0_0_, item0_.available as availabl3_0_0_, item0_.description as descript4_0_0_, item0_.name as name5_0_0_, item0_.owner_id as owner_id8_0_0_, item0_.request as request6_0_0_, user1_.id as id2_0_1_, user1_.email as email7_0_1_, user1_.name as name5_0_1_ from base_model item0_ left outer join base_model user1_ on item0_.owner_id=user1_.id where item0_.id=? and item0_.dtype='Item' [42102-214]
2023-01-16 09:59:46 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select item0_.id as id2_0_0_, item0_.available as availabl3_0_0_, item0_.description as descript4_0_0_, item0_.name as name5_0_0_, item0_.owner_id as owner_id8_0_0_, item0_.request as request6_0_0_, user1_.id as id2_0_1_, user1_.email as email7_0_1_, user1_.name as name5_0_1_ from base_model item0_ left outer join base_model user1_ on item0_.owner_id=user1_.id where item0_.id=? and item0_.dtype='Item' [42102-214]
2023-01-16 09:59:46 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select item0_.id as id2_0_0_, item0_.available as availabl3_0_0_, item0_.description as descript4_0_0_, item0_.name as name5_0_0_, item0_.owner_id as owner_id8_0_0_, item0_.request as request6_0_0_, user1_.id as id2_0_1_, user1_.email as email7_0_1_, user1_.name as name5_0_1_ from base_model item0_ left outer join base_model user1_ on item0_.owner_id=user1_.id where item0_.id=? and item0_.dtype='Item' [42102-214]
2023-01-16 09:59:46 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select item0_.id as id2_0_0_, item0_.available as availabl3_0_0_, item0_.description as descript4_0_0_, item0_.name as name5_0_0_, item0_.owner_id as owner_id8_0_0_, item0_.request as request6_0_0_, user1_.id as id2_0_1_, user1_.email as email7_0_1_, user1_.name as name5_0_1_ from base_model item0_ left outer join base_model user1_ on item0_.owner_id=user1_.id where item0_.id=? and item0_.dtype='Item' [42102-214]
2023-01-16 09:59:46 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select user0_.id as id2_0_0_, user0_.email as email7_0_0_, user0_.name as name5_0_0_ from base_model user0_ where user0_.id=? and user0_.dtype='User' [42102-214]
2023-01-16 09:59:46 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select item0_.id as id2_0_, item0_.available as availabl3_0_, item0_.description as descript4_0_, item0_.name as name5_0_, item0_.owner_id as owner_id8_0_, item0_.request as request6_0_ from base_model item0_ where item0_.dtype='Item' [42102-214]
2023-01-16 09:59:46 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select item0_.id as id2_0_, item0_.available as availabl3_0_, item0_.description as descript4_0_, item0_.name as name5_0_, item0_.owner_id as owner_id8_0_, item0_.request as request6_0_ from base_model item0_ where item0_.dtype='Item' [42102-214]
2023-01-16 09:59:46 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select item0_.id as id2_0_, item0_.available as availabl3_0_, item0_.description as descript4_0_, item0_.name as name5_0_, item0_.owner_id as owner_id8_0_, item0_.request as request6_0_ from base_model item0_ where item0_.dtype='Item' [42102-214]
2023-01-16 09:59:46 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select item0_.id as id2_0_0_, item0_.available as availabl3_0_0_, item0_.description as descript4_0_0_, item0_.name as name5_0_0_, item0_.owner_id as owner_id8_0_0_, item0_.request as request6_0_0_, user1_.id as id2_0_1_, user1_.email as email7_0_1_, user1_.name as name5_0_1_ from base_model item0_ left outer join base_model user1_ on item0_.owner_id=user1_.id where item0_.id=? and item0_.dtype='Item' [42102-214]
2023-01-16 09:59:46 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select item0_.id as id2_0_, item0_.available as availabl3_0_, item0_.description as descript4_0_, item0_.name as name5_0_, item0_.owner_id as owner_id8_0_, item0_.request as request6_0_ from base_model item0_ where item0_.dtype='Item' [42102-214]
2023-01-16 09:59:46 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select item0_.id as id2_0_, item0_.available as availabl3_0_, item0_.description as descript4_0_, item0_.name as name5_0_, item0_.owner_id as owner_id8_0_, item0_.request as request6_0_ from base_model item0_ where item0_.dtype='Item' [42102-214]
2023-01-16 09:59:46 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select item0_.id as id2_0_0_, item0_.available as availabl3_0_0_, item0_.description as descript4_0_0_, item0_.name as name5_0_0_, item0_.owner_id as owner_id8_0_0_, item0_.request as request6_0_0_, user1_.id as id2_0_1_, user1_.email as email7_0_1_, user1_.name as name5_0_1_ from base_model item0_ left outer join base_model user1_ on item0_.owner_id=user1_.id where item0_.id=? and item0_.dtype='Item' [42102-214]
2023-01-16 09:59:46 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
select item0_.id as id2_0_, item0_.available as availabl3_0_, item0_.description as descript4_0_, item0_.name as name5_0_, item0_.owner_id as owner_id8_0_, item0_.request as request6_0_ from base_model item0_ where item0_.dtype='Item' [42102-214]
2023-01-16 09:59:50 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
insert into base_model (email, name, dtype) values (?, ?, 'User') [42102-214]
2023-01-16 10:02:25 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "BASE_MODEL" не найдена
Table "BASE_MODEL" not found; SQL statement:
insert into base_model (email, name, dtype) values (?, ?, 'User') [42102-214]
